### ISSUE TITLE
Feature request: import GPX tracks

### ISSUE THREAD

**Issue by kyegupov** [osmandapp/Osmand#3044]
It's not obvious how to add a third-party GPX track to the map: Configure Map -> GPX Tracks just says "no tracks in folder" and does not give any hint on how to put them there.

Maybe add a note to the "menu" window saying "To import GPX tracks, download and "Open" them using OsmAnd(+) application"?

**Comment by Dagefoerde** [comment]
I agree, that has troubled me today, too. The message actually specifies the "/tracks" folder, but that was a bit misleading until I found it. I guess usability could be improved here.

On that note, opening GPX files from an arbitrary folder did not work out for me in two instances:
1) One file, downloaded via Chrome, gave the option to open it with OsmAnd. OsmAnd opened it but did not load it. Instead, a non-specific error showed up, indicating that the GPX file could not be read.
2) Another file, received via bluetooth, led Android to claiming that there were no application that could open it.

... but both files worked perfectly fine as soon as I discovered the "<MAP PATH>/tracks" folder and copied the GPX files into it. :slightly_smiling_face: 

So I guess there are two items: 

- [ ] Improve the "no tracks in folder" hint (full path name, maybe even an option to find files and copy them into there?)
- [ ] Improve how OsmAnd accepts GPX opening intents.

That would *really* be nice to have :slightly_smiling_face:
